packages/cli/src/databases/entities/FeatureConfig.ts

```diff
@@ -1,32 +1,15 @@
 import { Column, ColumnOptions, Entity, PrimaryColumn } from 'typeorm';
-import * as config from '@/config';
-import { DatabaseType, IFeatureConfigDb } from '@/Interfaces';
-import type { LdapConfig } from '@/Ldap/types';
-
-function resolveDataType(dataType: string) {
-	const dbType = config.getEnv('database.type');
-
-	const typeMap: { [key in DatabaseType]: { [key: string]: string } } = {
-		sqlite: {
-			json: 'simple-json',
-		},
-		postgresdb: {
-			datetime: 'timestamptz',
-		},
-		mysqldb: {},
-		mariadb: {},
-	};
-
-	return typeMap[dbType][dataType] ?? dataType;
-}
+import { IFeatureConfigDb } from '../..';
```

Comment:
can you please change this 
```suggestion
import { IFeatureConfigDb } from '@/Interfaces';
```

we need to stop using imports from `.` and `..`, as those were the largest culprits behind circular dependencies.

Reply:
Done